packages/protocol/contracts-0.8/UsingRegistry8.sol

```diff
@@ -68,43 +57,7 @@ contract UsingRegistry is Ownable {
     emit RegistrySet(registryAddress);
   }
 
-  function getAccounts() internal view returns (IAccounts) {
```

Comment:
Just wondering, why does this remove a bunch of the getters?

Also, is there a reason this is modeled after the original UsingRegistry rather than the more efficient UsingRegistryV2?

Reply:
It's done like this so I didn't have to make the interfaces have permissive pragmas.

Regarding the UsingRegistryV2, I'm not sure, the first implementation used this one, but I'll take a look.

Reply:
but now thinking about this, we already have flexible pragmas and everything lands in the right folder, so I guess I could bring it back. Will try that, thanks for the comment.

Reply:
Just remembered why not all getters could are there, because Mento's ones can be used with 0.8.

It is not using UsingRegistryV2 because gasPriceMinimum is using UsingRegistry.